src/ResourceManager/PowerBIEmbedded/Commands.PowerBI.Test/Commands.PowerBI.Test.csproj

```diff
@@ -0,0 +1,229 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@selasagi any reason you are creating separate cmdlet and test projects for the new cmdlets? Why can't everything be contained in `Commands.PowerBIEmbedded.Test` and `Commands.Management.PowerBI`?

Reply:
I prefer to leave it separate because it will be easier later to remove the commands and the tests of the WorkspaceCollection